pkg/telemetry/telemetry.go

```diff
@@ -398,6 +405,33 @@ func agentStats(k8sClient k8s.Client, managedNamespaces []string) (string, inter
 	return "agents", stats, nil
 }
 
+func logstashStats(k8sClient k8s.Client, managedNamespaces []string) (string, interface{}, error) {
+	stats := map[string]int32{resourceCount: 0, podCount: 0, stackMonitoringLogsCount: 0,
+		stackMonitoringMetricsCount: 0, serviceCount: 0, pipelinesCount: 0, pipelinesRefCount: 0}
+
+	var logstashList logstashv1alpha1.LogstashList
+	for _, ns := range managedNamespaces {
+		if err := k8sClient.List(context.Background(), &logstashList, client.InNamespace(ns)); err != nil {
+			return "", nil, err
+		}
+
+		for _, ls := range logstashList.Items {
+			ls := ls
```

Comment:
remove it?

Reply:
This one was caught by the linter, and is related to loop variable semantics in go - there is a great summary of the issue (and proposal for "fixing" it in future versions of golang) [here](https://github.com/golang/go/discussions/56010). I don't actually think that that behaviour will bite us here, given what we are doing, but I'm still happy keeping this "odd" looking assignment in here and keep the linter satisfied